lib/movie_favorite_screen/screen/favorite_screen.dart

```diff
@@ -0,0 +1,156 @@
+      builder: (context) {
+        return BlocProvider(
+          create: (context) => FavoriteCubit(),
+          child: const FavoriteScreen(),
+        );
+      },
+    );
+  }
+
+  const FavoriteScreen({Key? key}) : super(key: key);
+
+  @override
+  State<FavoriteScreen> createState() => _FavoriteScreenState();
+}
+
+class _FavoriteScreenState extends State<FavoriteScreen> {
+  FavoriteCubit get favoriteCubit => BlocProvider.of<FavoriteCubit>(context);
+
+  @override
+  void initState() {
+    favoriteCubit.getFavorite();
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      drawer: const DrawerMenu(),
+      appBar: AppBar(
+        centerTitle: true,
+        title: const Text(_kTitle),
+      ),
+      body: SafeArea(
+        child: BlocBuilder<FavoriteCubit, FavoriteState>(
+          builder: (context, state) {
+            if (state is FavoriteLoading) {
+              return const Center(
+                child: CircularProgressIndicator(
+                  backgroundColor: AppColors.darkBlue,
+                  color: Colors.white,
+                ),
+              );
+            }
+            if (state is FavoriteSuccess) {
+              return Padding(
+                padding: const EdgeInsets.only(
+                  left: 6.0,
+                  top: 10,
+                ),
+                child: ListView.separated(
+                  separatorBuilder: (BuildContext context, int index) =>
+                      const Divider(),
+                  itemCount: state.listMovies.length,
+                  itemBuilder: (BuildContext context, int index) {
+                    final movie = state.listMovies[index];
+                    return MovieCard(
+                      title: movie.title,
+                      overview: movie.overview,
+                      posterPath: movie.posterPath,
+                    );
+                  },
+                ),
+              );
+            }
+            return const SizedBox();
+          },
+        ),
+      ),
+    );
+  }
+}
+
+class MovieCard extends StatelessWidget {
+  final String title;
+  final String? overview;
+  final String posterPath;
+  const MovieCard({
+    Key? key,
+    required this.title,
+    required this.overview,
+    required this.posterPath,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        SizedBox(
+          height: 200,
+          width: 140,
+          child: Ink.image(
+            image: posterPath.isNotEmpty
+                ? NetworkImage(posterPath)
+                : Image.asset(
+                    MovieImage.movieImage,
+                    fit: BoxFit.cover,
+                  ).image,
+          ),
+        ),
+        Column(
+          children: [
+            SizedBox(
+              width: 200,
```

Comment:
if size of device will be wide it will look stange
<img width="950" alt="image" src="https://user-images.githubusercontent.com/20111049/165494107-dc364417-864e-4cea-91b0-5ef65358d64f.png">